lms/djangoapps/mobile_api/badges/views.py

```diff
@@ -0,0 +1,73 @@
+"""
+Mobile API views for badges
+"""
+from edxval.views import MultipleFieldLookupMixin
+from opaque_keys.edx.keys import CourseKey
+from rest_framework import generics
+
+from badges.models import BadgeClass, BadgeAssertion
+from mobile_api.badges.serializers import BadgeClassSerializer, BadgeAssertionSerializer
+from mobile_api.utils import mobile_view
+from xmodule_django.models import CourseKeyField
+
+
+class BadgeClassDetail(MultipleFieldLookupMixin, generics.RetrieveAPIView):
+    """
+    Get the details of a specific badge class.
+    """
```

Comment:
Please add docstring for the views.
See the following as an example: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/course_api/views.py#L17